jenkins_pipelines/environments/common/pipeline-reference.groovy

```diff
@@ -0,0 +1,44 @@
+def run(params) {
+    timestamps {
+        deployed = false
+        env.resultdir = "${WORKSPACE}/results"
+        env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
+        env.common_params = "--outputdir ${resultdir} --tf ${params.tf_file} --gitfolder ${resultdir}/sumaform"
+        try {
+            stage('Clone sumaform and susemanager-ci') {
```

Comment:
It seems it's not just cloning sumaform through terracumber, but also doing extra steps related to terraform files.
It might be good to clarify it on the title or even split the pure cloning of repos stage and other steps in different stages.

Reply:
In fact I think the title is a typo. It should say "Clone terracumber" ;-)

Nice catch.

About the split: I thought about it. Problem is that I am not sure it adds value, and creates an extra step and makes the pipeline "bigger".

Reply:
OK, fine with the renaming

Reply:
In fact I was half wrong.

The correct title for this is `Clone terracumber, susemanager-ci and sumaform` as it's exactly what it does. First terracumber, then susemanager-ci (for the tf files) and then with both it clones/updates sumaform.

Reply:
Fixed. About using separate steps, to be clarified and if agreed to be done on a separate PR.